db/scripts/media/populateMedia.js

```diff
@@ -1,13 +1,14 @@
 import db from "../../connection.js";
 
 const response = await db.query(
-   `INSERT INTO media (aws_key, media_title, media_descr, date, location) VALUES ($1, $2, $3, $4, $5);`,
+   `INSERT INTO media (aws_key, media_title, media_descr, date, latitude, logitude) VALUES ($1, $2, $3, $4, $5, %6);`,
```

Comment:
- change latitude/logitude to lat/lng
- should this info be in media table in the first place? Should we record loc_id instead?